### PR TITLE
feat: Audio blobs for merging samples.

### PR DESCRIPTION
**Pull request by noahdahlman**

### CODE REVIEW

File: components/StemPlayer.tsx [hunk 1]
```diff
@@ -99,7 +115,7 @@ const StemPlayer = (props: StemPlayerProps): JSX.Element => {
 			barGap: 2,
 		})
 		// Load audio from an XHR request
-		ws.load(details.audioHref)
+		ws.load_blob(blob)
```
Comment (drewcook) [inline components/StemPlayer.tsx:118]: 👍

File: contracts/Context.json [hunk 2]
```diff
@@ -478,8 +478,8 @@
     "version": "0.8.1+commit.df193b15.Emscripten.clang"
   },
   "networks": {},
-  "schemaVersion": "3.4.3",
-  "updatedAt": "2022-02-20T12:48:47.781Z",
+  "schemaVersion": "3.4.6",
+  "updatedAt": "2022-03-17T08:11:38.285Z",
```
Comment (drewcook) [inline contracts/Context.json:482]: We should preserve latest updated at, but also highest version on these `/contracts` changes.

File: pages/projects/[id].tsx [hunk 3]
```diff
@@ -369,7 +371,14 @@ const ProjectPage: NextPage<ProjectPageProps> = props => {
 		setSuccessMsg("Success! You've uploaded a new stem to this project and become a contributor.")
 		handleUploadStemClose()
 	}
-
+	const onNewFile = (newFile: Blob) => {
+		console.log('New File Found')
+		if (!files) {
+			setFiles(() => [newFile])
+		} else {
+			setFiles(files => [...files, newFile])
```
Comment (drewcook) [inline pages/projects/[id].tsx:379]: Can we ensure that we're only loading the same of any one sound here?  i.e. keep track in a hashmap or set of file cids, etc

Comment (noahdahlman) [reply pages/projects/[id].tsx:379]: Yeah I left a comment about that in the commit, but haven't handled it yet

File: components/StemPlayer.tsx [hunk 4]
```diff
@@ -78,15 +78,33 @@ type StemPlayerProps = {
...
 	useEffect(() => {
+		let blobFileToLoad: Blob | undefined
+		if (!blob) {
+			if (!loadingBlob) {
+				setLoadingBlob(true)
+				fetch(details.audioHref).then(resp => {
+					resp.blob().then(b => {
+						setBlob(b)
+						onNewFile(b)
+						blobFileToLoad = b
```
Comment (drewcook) [inline components/StemPlayer.tsx:101]: Needed to set the blob to a ref while we have access to it within this then block, so we can load it later on outside of this scope.

**Review by drewcook — APPROVED** [review]

**Pull request merged**